tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -390,11 +390,7 @@ install_image() {
 
 #Install guest image for confidential guests
 install_image_confidential() {
-	if [ "${ARCH}" == "s390x" ]; then
-		export MEASURED_ROOTFS=no
-	else
-		export MEASURED_ROOTFS=yes
-	fi
+	export MEASURED_ROOTFS=yes
```

Comment:
Wait why are you removing this check for s390x the measured rootfs is done differently. It needs to be no for s390x for the image.

Reply:
yeah, I have the same question. Let's turn off all measured rootfs on s390x.

Reply:
Changed!